<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>930</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Othello</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>180</y>
     <width>490</width>
     <height>490</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="player1_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>131</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>26</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>PLAYER 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="player1_image">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>130</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>131</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Player1 image</string>
   </property>
  </widget>
  <widget class="QLabel" name="player2_name">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>131</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>26</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>PLAYER 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="player2_image">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>52</y>
     <width>130</width>
     <height>80</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>131</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Player2 image</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_label">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>20</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>91</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>26</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>SCORE</string>
   </property>
  </widget>
  <widget class="QLabel" name="player1_score">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>31</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="score_label_2">
   <property name="geometry">
    <rect>
     <x>392</x>
     <y>70</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Carlito</family>
     <pointsize>40</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="player2_score">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>31</width>
     <height>70</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>31</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>690</y>
     <width>151</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
